example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -17,10 +17,6 @@ const {
   buildAndExecuteWorkflow,
   LambdaStep,
   conceptExists,
-  addProviders,
-  cleanupProviders,
-  addCollections,
-  cleanupCollections,
   getOnlineResources
 } = require('@cumulus/integration-tests');
 const { api: apiTestUtils } = require('@cumulus/integration-tests');
```

Comment:
We can't add this to the line above if they're coming from the same place?

Reply:
Good catch. This is probably the result of a bad merge or something